manifests/config.pp

```diff
@@ -3,14 +3,15 @@
 # Install confluence, See README.md for more.
 #
 class confluence::config(
-  $tomcat_port         = $confluence::tomcat_port,
-  $tomcat_max_threads  = $confluence::tomcat_max_threads,
-  $tomcat_accept_count = $confluence::tomcat_accept_count,
-  $tomcat_proxy        = $confluence::tomcat_proxy,
-  $tomcat_extras       = $confluence::tomcat_extras,
-  $manage_server_xml   = $confluence::manage_server_xml,
-  $context_path        = $confluence::context_path,
-  $ajp                 = $confluence::ajp,
+  $tomcat_port                     = $confluence::tomcat_port,
+  $tomcat_max_threads              = $confluence::tomcat_max_threads,
+  $tomcat_accept_count             = $confluence::tomcat_accept_count,
+  $tomcat_proxy                    = $confluence::tomcat_proxy,
+  $tomcat_extras                   = $confluence::tomcat_extras,
+  $tomcat_jdbc_settings            = $confluence::tomcat_jdbc_settings,
+  $manage_server_xml               = $confluence::manage_server_xml,
+  $context_path                    = $confluence::context_path,
+  $ajp                             = $confluence::ajp,
 ) {
 
```

Comment:
Sorry, at one point I had a longer variable name, did an align-regexp, and then didn't re-align. I can fix that.